tensorflow/lite/micro/tools/make/targets/stm32f4_makefile.inc

```diff
@@ -78,13 +78,14 @@ EXCLUDED_SRCS := \
 THIRD_PARTY_CC_SRCS := $(filter-out $(EXCLUDED_SRCS), $(THIRD_PARTY_CC_SRCS))
 MICROLITE_CC_SRCS := $(filter-out $(EXCLUDED_SRCS), $(MICROLITE_CC_SRCS))
 
+# Excludes micro_allocator_test because it calls CreateQuantizedFlatbufferTensor,
+# which use std::vector constructor which then invokes new.
+# Excludes memory_arena_threshold_test because the size difference of some
+# allocator classes between different architectures.
 # TODO(b/158324045): Examine why some tests fail here.
```

Comment:
let's change this TODO to http://b/158651472 since b/158324045 is about recording_micro_allocator_test which this PR is fixing.

And close b/158324045?

Reply:
Agree. Done.